src/sentry/api/endpoints/organization_auth_providers.py

```diff
@@ -20,9 +20,8 @@ def get(self, request: Request, organization) -> Response:
         :auth: required
```

Comment:
Not your code but the docstring for this class has a typo [here](https://github.com/getsentry/sentry/blob/11d161ad5ed49a0aab632231217a357c6bf3cc76/src/sentry/api/endpoints/organization_auth_providers.py#L19) and has a confusing first sentence, it's more readable as "List auth providers that are available to use for an organization" IMO. Your call if you want to change it!